app/presenters/publishing_api/world_location_news_presenter.rb

```diff
@@ -2,16 +2,19 @@ module PublishingApi
   class WorldLocationNewsPresenter
     include FeaturedDocumentsPresenter
 
-    attr_accessor :world_location, :update_type
+    attr_accessor :world_location, :world_location_news, :update_type
 
-    def initialize(world_location, update_type: nil)
-      self.world_location = world_location
+    def initialize(world_location_news, update_type: nil)
```

Comment:
suuuuper minor and I know it was done this way already, but what's the advantage of passing in `nil` here rather than `"major"`, would mean line 10 could be `self.update_type = update_type`

Reply:
We looked into this offline and found it isn't possible.  The `update_type` will always be `major`, as this is non-editionable content, so we think further work should be done (separately) to refactor the non-editionable presenters so this value isn't needed to be defined at all.